### PR TITLE
Update Config Value

### LINKED ISSUE #188: Read config value from x variable instead of y
Change `this_config["y"]` to ` this_config["x"]`
https://github.com/opensource9ja/danfojs/blob/e15aff19a58598d8f3e6da14426b594219418146/danfojs-browser/src/plotting/plot.js#L414

### PR DESCRIPTION
**Pull request by adithyaakrishna**
Signed-off-by: Adithya Krishna <aadithya794@gmail.com>

This PR Fixes #188

### CODE REVIEW

File: danfojs-browser/src/plotting/plot.js [hunk 1]
```diff
@@ -411,7 +411,7 @@ export class Plot {
         }
       });
 
-      trace['x'] = this.ndframe[this_config['y']].values;
+      trace['x'] = this.ndframe[this_config["x"]].values;
```
Comment (risenW) [inline danfojs-browser/src/plotting/plot.js:414]: @adithyaakrishna Thanks for the fix! One small comment before I merge, use a single quote inline with the existing style.

**Review by risenW — CHANGES_REQUESTED** [review]

File: danfojs-browser/src/plotting/plot.js (hunk 1 above)
Comment (adithyaakrishna) [reply danfojs-browser/src/plotting/plot.js:414]: My bad, Instead of single quotes, I gave double :/
Will fix this ASAP

**Comment by adithyaakrishna** [comment]
@risenW Is it possible if I can fix another issue in the same PR?
https://github.com/opensource9ja/danfojs/issues/77

As both of them are in the same files

**Review by risenW — APPROVED** [review]

**Comment by risenW** [comment]
> @risenW Is it possible if I can fix another issue in the same PR?
> #77
> 
> As both of them are in the same files

Sure, you can do it. I see you did it already 👍

**Pull request merged**